tailwind.config.js

```diff
@@ -0,0 +1,23 @@
+  // tailwind.config.js
+  module.exports = {
+
+    purge: [],
```

Comment:
Purge is appearing twice, you cannot have two properties of the same key, fortunately the latter one will take precedence over the former.